src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
I don't understand your comment.  I find having a separate uap configuration useful especially since it creates the 'uap' folder in the bin directory separate from the 'netstandard' folder when binplacing the binaries running the tests.